ci/plugins/mzcompose/hooks/command

```diff
@@ -25,7 +25,7 @@ if read_list BUILDKITE_PLUGIN_MZCOMPOSE_ARGS; then
     done
 fi
 
-echo "system uptime: $(cut -f1 -d' ' /proc/uptime)"
+date +"%Y-%m-%d %H:%M:%S" > step_start_timestamp
```

Comment:
How do we handle timezones btw? Is everything set to UTC? Something like `date --rfc-3339=seconds` would show timezone too for clarity:
```
2023-02-27 12:51:29+01:00
```

Reply:
Yes unfortunately `journalctl --since` does not parse time zones, I tried. So we rely on the fact that both the production and the consumption of the timestamp happen on the same machine.

Reply:
From what I have seen, Buildkite has no notion of time zones, so the time zone is whatever the time zone of the EC2 instance is, meaning US Eastern time in thie case.